src/test/datascience/export/fileConverter.vscode.web.test.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { anything, instance, mock, verify, when } from 'ts-mockito';
+import * as sinon from 'sinon';
+import { Uri } from 'vscode';
+import { IApplicationShell } from '../../../platform/common/application/types';
+import { IConfigurationService, IWatchableJupyterSettings } from '../../../platform/common/types';
+import { ExportFileOpener } from '../../../platform/export/exportFileOpener';
+import { FileConverter } from '../../../platform/export/fileConverter.web';
+import { IExport, IExportDialog, ExportFormat } from '../../../platform/export/types';
+
+suite('DataScience - File Converter', () => {
+    let fileConverter: FileConverter;
+    let exportPythonPlain: IExport;
+    let filePicker: IExportDialog;
+    let appShell: IApplicationShell;
+    let exportFileOpener: ExportFileOpener;
+    let configuration: IConfigurationService;
+    let settings: IWatchableJupyterSettings;
+    setup(async () => {
+        filePicker = mock<IExportDialog>();
+        exportPythonPlain = mock<IExport>();
+        appShell = mock<IApplicationShell>();
+        exportFileOpener = mock<ExportFileOpener>();
+        configuration = mock<IConfigurationService>();
+        settings = mock<IWatchableJupyterSettings>();
+        when(configuration.getSettings(anything())).thenReturn(instance(settings));
+        when(settings.pythonExportMethod).thenReturn('direct');
+        when(filePicker.showDialog(anything(), anything(), anything())).thenReturn(
+            Promise.resolve(Uri.file('test.py'))
+        );
+        when(appShell.showErrorMessage(anything())).thenResolve();
+        when(exportPythonPlain.export(anything(), anything(), anything())).thenResolve();
+        when(filePicker.showDialog(anything(), anything())).thenResolve(Uri.file('foo'));
+        when(exportFileOpener.openFile(anything(), anything(), anything())).thenResolve();
+        fileConverter = new FileConverter(

```

Comment:
The `FileConverter` on Web now differs from the Nodejs one. Not sure if it stays so after we migrate most features to Web, which depends on the implementation details of Export actions.